README.md

```diff
@@ -1,3 +1,17 @@
+# Kubernetes GPU as an app in the Giant Swarm Playground Catalog
+
+**What is it?**
+It is a recipe to install the GPU drivers into our nodes to let customers run GPU workloads.
+
+**Why did we add it?**
+So far, customers have needed to run it by themselves. We've had to sync all time every time there is a change. Now we can provide a new release when a new driver version is out (and tested) and they can easily install it.
```

Comment:
```suggestion
Users that want to run GPU workloads, e.g. for AI and similar use cases need to have special drivers (CUDA) installed on the GPU machines. This app makes sure those drivers are present and installed in the right version.
```

I think the above is kinda generic to all apps we provide, also wanted to make this less about customers but more about users (as this app is public and everyone could use it)